models/admin/domains_proc.sql

```diff
@@ -11,7 +11,7 @@ FROM (
 	google_search_console_account search_console_account,
 	google_analytics_account,
 	time_of_entry,
-	first_value(time_of_entry) OVER (PARTITION BY site_name ORDER BY time_of_entry DESC) lv
+	first_value(time_of_entry) OVER (PARTITION BY site_name,google_analytics_account,google_search_console_account ORDER BY time_of_entry DESC) lv
```

Comment:
Not sure we want to partition at this granularity - we want to pull the latest value for the site as a whole (with this GA account / GSC account partition, if we update the account name it'd create two rows for the site).